sda-commons-web-autoconfigure/src/main/resources/org/sdase/commons/spring/boot/web/defaults.properties

```diff
@@ -1,3 +1,4 @@
 server.servlet.context-path=/api
 server.port=8080
-server.error.include-message=always
+server.max-http-header-size=8192
```

Comment:
Isn't that already the default value? 

> The actual default value for Tomcat and Jetty is 8kB, and the default value for Undertow is 1MB.

https://www.baeldung.com/spring-boot-max-http-header-size

Reply:
Yes. I defined it, because it's not always 8kB. Do you want to omit the setting because we use Tomcat? Imho we set it for Dropwizard as well although the default is also 8kB.

Reply:
then let's leave it as it is to not rely on defaults of other libraries.